### PR TITLE
Add Shuttlecloud user interview

### PR DESCRIPTION
**Pull request by brian-brazil**

### CODE REVIEW

File: content/blog/2016-09-01-interview-with-shuttlecloud.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+By integrating our API into their offerings, our customers allow their users to easily migrate their email and contacts from one participating provider to another, reducing the friction users face when switching to a new provider. The 24/7 email providers supported include all major US internet service providers: Comcast, Time Warner Cable, AT&T, Verizon, and more.
+
+By offering end users a simple path for migrating their emails (while keeping complete control over the import tool’s UI), our customers dramatically improve user activation and onboarding.
+
+![ShuttleCloud's integration with Gmail](/assets/blog/2016-09-07/gmail-integration.png)
+***ShuttleCloud’s [integration](https://support.google.com/mail/answer/164640?hl=en) with Google’s Gmail Platform.*** *Gmail has imported data for 3 million users with our API.*
+
+
+ShuttleCloud’s technology, encrypts all the data required to process an import, in addition to following the most secure standards (SSL, oAuth) to ensure the confidentiality and integrity of API requests. Our technology allows us to guarantee our platform’s high availability, with up to 99.5% uptime assurances.
```
Comment (beorn7) [inline content/blog/2016-09-01-interview-with-shuttlecloud.md]: 1st comma is spurious.

File: content/blog/2016-09-01-interview-with-shuttlecloud.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+![Initial Shuttlecloud System](/assets/blog/2016-09-07/Prometheus-System-1.jpg)
+
+   * The source of all necessary data is a status database in a CouchDB. There, each document represents one status of an operation. This information is processed by the Status Importer and stored in a relational manner in a MySQL database.
+
+   * A component gathers data from that database, with the information aggregated and post-processed into several views. 
+      * One of the views is the email report, which we needed for reporting purposes. This is sent via email. 
+      * The other view pushes data to a dashboard, where it can be easily controlled. The dashboard service we used was external. We trusted Ducksboard, not only because the dashboards were easy to set up and looked beautiful, but also because they provided automatic alerts if a threshold was reached.
+
+With all that in place, it didn’t take us long to realize that we would need a proper metrics, monitoring, and alerting system as the number of projects started to increase.
```
Comment (beorn7) [inline content/blog/2016-09-01-interview-with-shuttlecloud.md]: It's "alerting system" here, but then "alert system" multiple times further down. I suggest to call it "alerting system" consistently.

File: content/blog/2016-09-01-interview-with-shuttlecloud.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+We decided to export and monitor the following metrics:
+
+   * `operation_requests_total`
+   * `operation_statuses_total`
+   * `operation_errors_total`
+
+![Shuttlecloud Prometheus System](/assets/blog/2016-09-07/Prometheus-System-2.jpg)
+
+We have most of our services duplicated in two Google Cloud Platform availability zones. That includes the monitoring system. It’s straightforward to have more than one operation exporter in two or more different zones, as Prometheus can aggregate the data from all of them and make one metric (i.e., the maximum of all). We currently don’t have Prometheus or the Alertmanager in HA—only a metamonitoring instance—but we are working on it.
```
Comment (beorn7) [inline content/blog/2016-09-01-interview-with-shuttlecloud.md]: Personally, I much prefer spaces around the (em- or en-) dashes.

**Comment by beorn7** [comment]
Only nits. :+1: in general.

File: content/blog/2016-09-01-interview-with-shuttlecloud.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+      * `http_requests_total{job="my_super_app_1",environment="staging"}` - the time series corresponding to the staging environment for the app "my\_super\_app_1".
+      * `http_requests_total{job="my_super_app_1"}` - the time series for all environments for the app "my\_super\_app\_1".
+      * `http_requests_total{environment="staging"}` - the time series for all staging environments for all jobs.
+   * Prometheus supports a DNS service for service discovery. We happened to already  have an internal DNS service.
+   * There is no need to install any external services (unlike Sensu, for example, which needs a data-storage service like Redis and a message bus like RabbitMQ). This might not be a deal breaker, but it definitely makes the test easier to perform, deploy, and maintain.
+   * Prometheus is quite easy to install, as you only need to download an executable Go file. The docker container also works well and it is easy to start.
```
Comment (juliusv) [inline content/blog/2016-09-01-interview-with-shuttlecloud.md]: docker -> Docker

File: content/blog/2016-09-01-interview-with-shuttlecloud.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+Initially we were only using some metrics provided out of the box by the [node_exporter](https://github.com/prometheus/node_exporter), including:
+
+   * hard drive usage.
+   * memory usage.
+   * if an instance is up or down.
+
+Our internal DNS service is integrated to be used for service discovery, so every new instance is automatically monitored.
+
+Some of the metrics we used, which were not provided by the node_exporter by default, were exported using the [node_exporter textfile collector](https://github.com/prometheus/node_exporter#textfile-collector) feature. The first alerts we declared on the Prometheus Alertmanager were mainly related to the operation metrics mentioned above.
```
Comment (juliusv) [inline content/blog/2016-09-01-interview-with-shuttlecloud.md]: "operation metrics" -> "operational metrics"?

Comment (juliusv) [inline content/blog/2016-09-01-interview-with-shuttlecloud.md]: Ah, maybe "operation metrics" is right after all, if it's about their business operations...

**Comment by juliusv** [comment]
:+1: besides mine and @beorn7's nits.

File: content/blog/2016-09-01-interview-with-shuttlecloud.md (hunk 1 above)
Comment (brian-brazil) [inline content/blog/2016-09-01-interview-with-shuttlecloud.md]: Removed

File: content/blog/2016-09-01-interview-with-shuttlecloud.md (hunk 2 above)
Comment (brian-brazil) [inline content/blog/2016-09-01-interview-with-shuttlecloud.md]: Done

File: content/blog/2016-09-01-interview-with-shuttlecloud.md (hunk 4 above)
Comment (brian-brazil) [inline content/blog/2016-09-01-interview-with-shuttlecloud.md]: Done

File: content/blog/2016-09-01-interview-with-shuttlecloud.md (hunk 5 above)
Comment (brian-brazil) [inline content/blog/2016-09-01-interview-with-shuttlecloud.md]: Operation metrics would be about one operation, so changed.

File: content/blog/2016-09-01-interview-with-shuttlecloud.md (hunk 3 above)
Comment (brian-brazil) [inline content/blog/2016-09-01-interview-with-shuttlecloud.md]: Yes, in this usage there should be spaces.

**Pull request merged**